src/implementations/twig/components/media-container/media-container.html.twig

```diff
@@ -93,44 +106,51 @@
 
 {# Print the result #}
 
-<figure class="{{ _css_class }}"{{ _extra_attributes|raw }}>
-{% if _embedded_media is not empty %}
-  {% set _embed_class = _ratio ? "ecl-media-container__media ecl-media-container__media--ratio-" ~ _ratio : "ecl-media-container__media" %}
-  <div class="{{_embed_class}}">
-    {%- block embedded_media _embedded_media -%}
-  </div>
-{% elseif _sources is not empty and _sources is iterable %}
-  <video
-    class="ecl-media-container__media"
-    controls
-    poster="{{ _image }}"
-  >
-{% for source in _sources %}
-  <source type="{{ source.type }}" src="{{ source.src }}" />
-{% endfor %}
-  {% if _tracks is not empty and _tracks is iterable %}
-  {% for track in _tracks %}
-    <track
-      kind="{{ track.kind }}"
-      label="{{ track.label }}"
-      src="{{ track.src }}"
-      srclang="{{ track.src_lang }}"
-    />
+<div class="{{ _css_class }}"{{ _extra_attributes|raw }}>
+  <figure class="{{ _figure_class }}">
+  {% if _embedded_media is not empty %}
+    <div class="{{_embed_class}}">
+      {%- block embedded_media _embedded_media -%}
+    </div>
+  {% elseif _sources is not empty and _sources is iterable %}
+    <video
+      class="ecl-media-container__media"
+      controls
+      poster="{{ _image }}"
+    >
+  {% for source in _sources %}
+    <source type="{{ source.type }}" src="{{ source.src }}" />
   {% endfor %}
+    {% if _tracks is not empty and _tracks is iterable %}
+    {% for track in _tracks %}
+      <track
+        kind="{{ track.kind }}"
+        label="{{ track.label }}"
+        src="{{ track.src }}"
+        srclang="{{ track.src_lang }}"
+      />
+    {% endfor %}
+    {% endif %}
+    </video>
+  {% else %}
+    {% include '@ecl/picture/picture.html.twig' with {
+      picture: _picture,
+      extra_classes: 'ecl-media-container__picture',
+      extra_image_classes: 'ecl-media-container__media',
+    } only %}
   {% endif %}
-  </video>
-{% else %}
-  {% include '@ecl/picture/picture.html.twig' with {
-    picture: _picture,
-    extra_classes: 'ecl-media-container__picture',
-    extra_image_classes: 'ecl-media-container__media',
-  } only %}
-{% endif %}
-{% if _description is not empty %}
-  <figcaption class="ecl-media-container__caption">
-    {{- _description -}}
-  </figcaption>
-{% endif %}
-</figure>
+  {% if _description is not empty %}
+    <figcaption class="ecl-media-container__caption">
+      {{- _description -}}
+    </figcaption>
+  {% endif %}
+  </figure>
+
+  {% if _expandable is not empty %}
+    {% include '@ecl/expandable/expandable.html.twig' with _expandable|merge({
+      extra_classes: _expandable_class,
```

Comment:
doing this would override existing extra_classes in the data, i think we should merge this class with the others, if present.